allennlp/nn/beam_search.py

```diff
@@ -381,3 +409,178 @@ def _search(
         all_predictions = torch.cat(list(reversed(reconstructed_predictions)), 2)
 
         return all_predictions, last_log_probabilities
+
+    @classmethod
+    def without_sampling(
+        cls,
+        end_index: int,
+        max_steps: int = 50,
+        beam_size: int = 10,
+        per_node_beam_size: int = None,
+        sampler: Sampler = None,
+    ) -> "BeamSearch":
+        """
+        Given an index of the end token in target vocabulary, return a BeamSearch object
+        that can be used to find `beam_size` candidate sequences, found by sampling from
+        the top `k` tokens to choose from at each step.
```

Comment:
```suggestion
        Given an index of the end token in target vocabulary, return a `BeamSearch` object
        that can be used to find `beam_size` candidate sequences.
```

Reply:
Looks like this change didn't go through

Reply:
Weird it looks right on my local branch, maybe I forgot to add that file or something